### DISCUSSION TITLE
Automatically upload pasted media to Curator

### DISCUSSION THREAD

**Discussion by DanielSpravtsev** [awcodes/filament-tiptap-editor#364] [Ideas]
Hey @awcodes!
Is there a way to automatically upload pasted media (at least images) to Curator media library?

**Comment by DanielSpravtsev** [comment]
@awcodes Maybe there is a way to hook into TiptapEditor content change and if there `<img>` tag, then download media from src attribute (copy title, alt and etc)  and replace it with local storage path 

or maybe we can just hook into tiptap itself and make [something like this](https://stackoverflow.com/questions/78147060/how-to-upload-inserted-images-with-image-extension-in-tiptap)

**Comment by awcodes** [comment]
I'll have to think about this. It's been brought up before, but I'm not totally sold on it.

If you can come up with a solid PR for it, I'd be happy to review it and explore it more.
